src/components/apiDelegation/given/OverviewPage/OverviewPage.tsx

```diff
@@ -17,7 +19,7 @@ export const ApiDelegationOverviewPage = () => {
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.api_delegations')}</PageHeader>
         <PageContent>
           <div className={classes.pageContent}>
-            <OrgDelegationOverviewPageContent />
+            <OverviewPageContent layout={LayoutState.Given} />
```

Comment:
Remove setting layout in component now as it is only needed to be set in overviewOrgSlice.

Do this instead on line 14: dispatch(setLayout(layout));
```suggestion
            <OverviewPageContent />
```